src/test/java/core/basesyntax/PlaneTest.java

```diff
@@ -41,7 +41,7 @@ public void checkThatBuilderClassIsPresent() {
         try {
             planeBuilderClass.get().getMethod("build");
         } catch (NoSuchMethodException e) {
-            Assert.fail("Builder class should contain method 'build'");
+            Assert.fail("PlaneBuilder class should contain method 'build'");
```

Comment:
don't modify test class

Reply:
okey

Reply:
return this file to the original state and make a commit. It should be gone from this PR

Reply:
done